go.mod

```diff
@@ -39,3 +41,5 @@ require (
 	k8s.io/client-go v0.0.0-20190620085101-78d2af792bab
 	k8s.io/utils v0.0.0-20191010214722-8d271d903fe4 // indirect
 )
+
+replace github.com/layer5io/service-mesh-performance-specification v0.0.0-20200617124327-27b13cd1be5e => github.com/kanishkarj/service-mesh-performance-specification v0.0.0-20200620172814-ca391dc349b7
```

Comment:
@kanishkarj Is this just for test purpose ?

Reply:
Yes, until we merge the changes in the SMPS repo.